addons/web/static/src/views/fields/statusbar/statusbar_field.scss

```diff
@@ -52,11 +52,11 @@
             }
 
             &.o_arrow_button_current.disabled {
-                background-color: $o-gray-100;
-                color: o-color(odoo);
+                background-color: $o-brand-odoo;
```

Comment:
goes oposite of https://github.com/odoo/odoo/commit/8ad60c1ea6a132803533f6471428fa52ef503039#diff-524bd164ef6856a7e8bbb24c6b1fa29834a5aa07489cc6e9fae9b5334620dc07R54 7 days ago